scanner/main.go

```diff
@@ -75,7 +83,7 @@ func (flags *excludeFlags) Set(value string) error {
 
 func (flags excludeFlags) Has(path string) bool {
 	for _, exclude := range flags {
-		if path == exclude {
+		if trimTrailingSlashes(path) == trimTrailingSlashes(exclude) {
```

Comment:
Please clean up the path in `Set()`

Reply:
Done, but I also had to add a `filepath.Clean` in the `filepath.Walk` function